test/integration/features/proxy.feature

```diff
@@ -0,0 +1,81 @@
+@proxy @minishift-only
```

Comment:
@hferentschik In current form which does not block all connections but more it checks the log, the minishift specific part lies in checks for download of OpenShift images, for CDK it needs to check for RedHat's registry.